File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.java
class Solution {
    public String reversePrefix(String word, char ch) {
        StringBuilder sb = new StringBuilder();
        String sub = word.substring(0,word.indexOf(ch)+1);

        sb.append(sub);
        sb.reverse();
        sb.append(word.substring(word.indexOf(ch)+1,word.length()));
        return sb.toString();
    }
}

File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.py
class Solution:
    def reversePrefix(self, word: str, ch: str) -> str:
        i=word.find(ch)
        if i==-1:
            return word
        else:
            q = word[:i+1]
            p = word[i+1:]
            q = q[::-1]
            return q+p
        